### DISCUSSION TITLE
GramJam Fall 2022 Final Submission Thread

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#51] [News + Announcements]
As a reply to this post, submit your project using the following template:

### Description
{{Text for short description}}

**Link to code:**
**Link(s) to additional resources:** {{mock-ups, slides, blog post, video demo, live demo, etc.}}

### Team Name:
- teammate names in a list
------------------------
The completed submission should look like this:

### Description
A Game by Nikola Whallon with Transcription powered-playing made possible with Deepgram.  The game is simple. Outrun the ghosts and vanquish them with "fire!", "lightning!", and "ice!" when they get too close.

**Link to code:** https://github.com/deepgram/SpookySpeechSpells
**Additional Resources:** live demo -  https://spookyspeechspells.deepgram.com/

### Spooky Spells:
- Nikola Whallon
- Natalie Rutgers

**Comment by jasonmajors** [comment]
## NoiceMail
### Description
A service that converts voicemails to text messages with the option to include full transcript, summary, topics, sentiment, and language.

**Link to code**: https://github.com/deepgram-devs/noicemail 
**Additional Resources**: 
* Sign up: https://noicemail.deepgram.com/
* Mockups: https://www.figma.com/file/bq1U7Q1anw6xoIzM99JK4I/Noicemail?node-id=0%3A1
* Slides: https://docs.google.com/presentation/d/11Pug0EtWknKsnwkRUXi4hKrFcrGRIE30ZuS6-GEsdk4/edit#slide=id.g162dcf642b8_0_570

## Your Mom
* Andrew Halle
* Joke Lohrey
* Kusum Ijari
* Jason Majors
* Kat McMillan
* Pankaj Trivedi
* Nikola Whallon

**Reply by BekahHW** [reply]
My mom had nothing to do with this project.

**Comment by jcdyer** [comment]
## N!ghtm∆r3 Sl!d3sh0w
### Description
Boring Slideshows? No!  Make your next slideshow an experience that will be seared into your team's brain and live there rent-free forever.

Link to code: https://github.com/deepgram-devs/gramjam2022-lucky-number-s3v3n.git

Link(s) to additional resources: https://boringslideshows.no.deepgram.com/

## Lucky Number S3V3N
* Cliff Dyer
* Desmond Caulley
* Jordan Krugman
* Martavis Parker
* Shir Goldberg
* Steven Risch
* Tivo Gonzalez

**Comment by ghost** [comment]
2022: The Game!
Description
Survive 2022!

https://github.com/deepgram-devs/2022-the-game

Link(s) to additional resources: (http://35.88.106.137:8080/app)

The Artists Formerly Known As Team 2
James Berry
Jeremy Cline
Alex Heyd
Jenny Draxl
Ingrid Elise Dorai-Rekka
Minh Li
Elliot Bearden
Evan Henry

**Comment by nikking-dg** [comment]
Speech Racer

A fast talking game that tests your speaking speed and accuracy.

https://speechracer.com

Link to code: https://github.com/deepgram-devs/speechracer
Link(s) to additional resources: https://docs.google.com/presentation/d/1LGiWLKV9AJKSu0GcDShGQgfy5vgfUkRoBGyF7ew5ZzA/edit?usp=sharing
Blog post: [https://dev.to/bekahhw/speech-racer-my-first-hackathon-project-2ona](https://dev.to/bekahhw/speech-racer-my-first-hackathon-project-2ona)
Twitter: [@Speedracer](https://twitter.com/speechracer)
![speechracer gif (900 × 900 px)](https://user-images.githubusercontent.com/34313413/202761090-ffc59220-abc5-4a00-8a75-f3b0d72087e8.gif)

Team Name: The Sandtraps

Alexa De La Torre
Beka Hawrot Weigel
Brian Hillis
Chris Foster
Nick Martin
Nik King

**Reply by natalierutgers** [reply]
The gif is so slick.
